cdap-standalone/src/main/resources/cdap-site.xml

```diff
@@ -275,4 +275,12 @@
     </description>
   </property>
 
+  <property>
+    <name>ui.feature.cdc</name>
```

Comment:
what about delta.default.checkpoint.directory?

Reply:
it's not strictly needed, but probably better to set so that it's contained within the same directory structure. Will add